lib/core/src/debug/monitor.rs

```diff
@@ -0,0 +1,603 @@
+use crate::prelude::*;
+
+use js_sys::ArrayBuffer;
+use js_sys::WebAssembly::Memory;
+use std::collections::VecDeque;
+use std::f64;
+use wasm_bindgen::JsCast;
+use wasm_bindgen::prelude::*;
+use wasm_bindgen;
+use web_sys::*;
+
+
+
+// ===============
+// === Helpers ===
+// ===============
+
+fn window() -> web_sys::Window {
+    web_sys::window().unwrap_or_else(|| panic!("Cannot access window."))
+}
+
+fn document() -> web_sys::Document {
+    window().document().unwrap_or_else(|| panic!("Cannot access window.document."))
+}
+
+fn body() -> web_sys::HtmlElement {
+    document().body().unwrap_or_else(|| panic!("Cannot access window.document.body."))
+}
+
+fn performance() -> Performance {
```

Comment:
These are rather laconic names for functions in module. Perhaps you want to follow "getters without get" rule, but I think these function are not plain getters, so could be namet "get_performance_info" or something like that

Reply:
Hmm, but this is `window.performance` getter. It is not "performance info", it is "performance object". I'm happy to change the name, but I dont like "get_performance_info" too much. What do you think?

Reply:
Ok, I see that "performance object" is something javasscript specific? If you think it will be clear in usage places, then leave it as it is.